test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionExecutorTests.cs

```diff
@@ -33,22 +33,6 @@ public class ControllerActionExecutorTests
         private delegate dynamic ReturnTaskAsDynamicValue(int i, string s);
 
         [Fact]
-        public async Task AsyncAction_WithVoidReturnType()
-        {
-            // Arrange
-            var methodWithVoidReturnType = new MethodWithVoidReturnType(TestController.VoidAction);
-
-            // Act
-            var result = await ControllerActionExecutor.ExecuteAsync(
-                                                        methodWithVoidReturnType.GetMethodInfo(),
-                                                        null,
-                                                        (IDictionary<string, object>)null);
-
-            // Assert
-            Assert.Same(null, result);
-        }
```

Comment:
FYI for anyone else, this test was removed because it was testing a static method, which we don't support anymore.